### ISSUE TITLE
Suggestion: /.well-known/llms.txt instead

### ISSUE THREAD

**Issue by simonw** [AnswerDotAI/llms-txt#2]
These days there's a increasingly adopted alternative to the `/robots.txt` and `/sitemap.xml` and `/favicon.ico` standards - the `/.well-known` prefix: https://en.wikipedia.org/wiki/Well-known_URI

These have been registered with IANA since ~2010: https://www.iana.org/assignments/well-known-uris/well-known-uris.xhtml

The RFC has the argument for why this is a good idea, which honestly I haven't read in a few years so I'm a bit rusty on: https://datatracker.ietf.org/doc/html/rfc5785

Oh, maybe this is it (quoting the RFC):

> When this happens, it is common to designate a "well-known location" for such data, so that it can be easily located. However, this approach has the drawback of risking collisions, both with other such designated "well-known locations" and with pre-existing resources.
>
> To address this, this memo defines a path prefix in HTTP(S) URIs for these "well-known locations", "/.well-known/". Future specifications that need to define a resource for such site-wide metadata can register their use to avoid collisions and minimise impingement upon sites' URI space.

**Comment by 3052** [comment]
100%. its kind of sad that a group of this site didn't bother doing any research on prior art before throwing this up.

**Comment by martinthomson** [comment]
A more complete argument can be found in [RFC 8820](https://datatracker.ietf.org/doc/html/rfc8820).  The original name of that document was a variation on "get off my lawn", which is a good summary of the general problem here.

**Comment by CodeFromAnywhere** [comment]
it's a great idea to use .well-known but it is still not so well known :(

**Comment by jeffbryner** [comment]
100% Here's examples of all the other .well-known's in use https://en.m.wikipedia.org/wiki/Well-known_URI#List_of_well-known_URIs

**Comment by NatoBoram** [comment]
`/.well-known/llms.md` would make more sense since it's a markdown file

**Comment by jfunction** [comment]
I agree. Looks like there are some hanging questions. Will we get a decision on this? Who decides, and who should be assigned?

**Comment by simonwfarrow** [comment]
I created specification for adding llms.txt and llms-full.txt to [well-known-uris](https://github.com/protocol-registries/well-known-uris). PR [here](https://github.com/AnswerDotAI/llms-txt/pull/44)
I reached out on Discord, and I think the author (@jph00) responded to say happy for someone to give it a go.

Next step is to submit the above spec as a provisional listing  - happy to do this as long as everyone agrees.

**Comment by jph00** [comment]
Go for it @simonwfarrow :)

**Comment by ite-klass** [comment]
> and also doing so would make it impossible to go to the next planned step, of allowing an llms.txt file to be placed at sub-path level.

I hope this doesn't mean they would potentially check for files on every path depth level - unnecessarily spamming servers with requests.

**Comment by rhues** [comment]
See also [#85](https://github.com/AnswerDotAI/llms-txt/issues/85)
